gateway/src/test/java/io/camunda/zeebe/gateway/api/signal/BroadcastSignalTest.java

```diff
@@ -47,5 +52,6 @@ public void shouldMapRequestAndResponse() {
     final SignalRecord brokerRequestValue = brokerRequest.getRequestWriter();
     assertThat(brokerRequestValue.getSignalName()).isEqualTo(request.getSignalName());
     MsgPackUtil.assertEqualityExcluding(brokerRequestValue.getVariablesBuffer(), variables);
+    assertThat(brokerRequestValue.getTenantId()).isEqualTo(request.getTenantId());
```

Comment:
🔧 This test is a bit interesting. It says it should map the request and response, yet it never asserts anything in the response, except that it's not null 😕